app/shared/src/services/__tests__/CurrencyFormatter.test.js

```diff
@@ -1,9 +0,0 @@
-// @flow
-
-import CurrencyFormatter from '../CurrencyFormatter';
-
-it('works', () => {
-  expect(CurrencyFormatter(10)).toBe('10');
-  expect(CurrencyFormatter(12.3)).toBe('12.3');
-  expect(CurrencyFormatter(72.8499999)).toBe('72.85');
```

Comment:
Please try not to delete tests like this. For example, this one is quite important because it was created after bugfix (fixing the .84999999 rounding). Could you revert it, please?

Reply:
Add them to `app/shared/src/currency/__tests__/CurrencyFormatter.test.js`